### PR TITLE
nn and graphs

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
in general, also the NAME and DESCRIPTION props, and the description token need to be updated.

File: braph2genesis/graphs/_MultigraphBUD.gen.m [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 %%% ¡description!
 In a binary undirected multigraph with fixed densities, all the layers 
- consist of binary undirected (BU) graphs derived from the same weighted 
+ consist of binary undirected graphs (MultigraphBUD) derived from the same weighted 
  connectivity matrix binarized at different densities.
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultigraphBUD.gen.m]: Should be:

In a binary undirected multigraph with fixed densities (MultigraphBUD), all the layers consist of binary undirected graphs (GraphBU) derived from the same weighted connectivity matrix binarized at different densities.

File: braph2genesis/graphs/_MultigraphBUT.gen.m [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 %%% ¡description!
 In a binary undirected multigraph with fixed thresholds, all the layers 
- consist of binary undirected (BU) graphs derived from the same 
+ consist of binary undirected graphs (MultigraphBUT) derived from the same 
  weighted connectivity matrix binarized at different thresholds.
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultigraphBUT.gen.m]: Same logic as in multigraphBUD

File: braph2genesis/graphs/_MultigraphBUD.gen.m (hunk 1 above)
Comment (iok-ui) [reply braph2genesis/graphs/_MultigraphBUD.gen.m]: or leave out "(GraphBU)", to be consistent with the description in other graph elements.

File: braph2genesis/graphs/_MultigraphBUD.gen.m [hunk 3]
```diff
@@ -121,12 +121,12 @@ consist of binary undirected (BU) graphs derived from the same weighted
...
 NAME (constant, string) is the name of the binary undirected multigraph with fixed densities.
 %%%% ¡default!
-'MultigraphBUD'
+'Multigraph Binary Undirected at fixed Densities'
 
 %%% ¡prop!
 DESCRIPTION (constant, string) is the description of the binary undirected multigraph with fixed densities.
 %%%% ¡default!
-'In a binary undirected multigraph with fixed densities, all the layers consist of binary undirected (BU) graphs derived from the same weighted connectivity matrix binarized at different densities. There are no connections between layers.'
+'In a binary undirected multigraph with fixed densities, all the layers consist of binary undirected graphs (MultigraphBUD) derived from the same weighted connectivity matrix binarized at different densities. There are no connections between layers.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultigraphBUD.gen.m]: Change according to the one in %%% ¡description!

File: braph2genesis/graphs/_MultigraphBUT.gen.m [hunk 4]
```diff
@@ -121,12 +121,12 @@ consist of binary undirected (BU) graphs derived from the same
...
 NAME (constant, string) is the name of the binary undirected multigraph with fixed thresholds.
 %%%% ¡default!
-'MultigraphBUT'
+'Multigraph Binary Undirected at fixed Thresholds'
 
 %%% ¡prop!
 DESCRIPTION (constant, string) is the description of the binary undirected multigraph with fixed thresholds.
 %%%% ¡default!
-'In a binary undirected multigraph with fixed thresholds, all the layers consist of binary undirected (BU) graphs derived from the same weighted connectivity matrix binarized at different thresholds. There are no connections between layers.'
+'In a binary undirected multigraph with fixed thresholds, all the layers consist of binary undirected graphs (MultigraphBUT) derived from the same weighted connectivity matrix binarized at different thresholds. There are no connections between layers.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultigraphBUT.gen.m]: Same logic as in multigraphBUD

File: braph2genesis/graphs/_MultilayerBUD.gen.m [hunk 5]
```diff
@@ -1,11 +1,11 @@
 %% ¡header!
 MultilayerBUD < MultilayerWU (g, multilayer binary undirected with fixed densities graph) is a binary undirected multilayer graph with fixed densities.
 %%% ¡description!
-In a multilayer binary undirected with fixed densities (BUD) graph, the layers 
- are those of binary undirected (BU) multilayer graphs derived from the same 
+In a multilayer binary undirected with fixed densities graph (MultilayerBUD), the layers 
+ are those of binary undirected multilayer graphs derived from the same
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultilayerBUD.gen.m]: Should be:

In a multilayer binary undirected **graph** with fixed densities (MultilayerBUD), ...

File: braph2genesis/graphs/_MultilayerBUD.gen.m [hunk 6]
```diff
@@ -144,12 +144,12 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+'In a multilayer binary undirected with fixed densities graph (MultilayerBUD), the layers are those of binary undirected multilayer graphs derived from the same weighted supra-adjacency matrix binarized at different densities. The supra-adjacency matrix has a number of partitions equal to the number of densities. Layerswithin the binary undirected multilayer graphs could have different number of nodes with within-layer binary undirected edges. Edges can be either 0 (absence of connection) or 1 (existence of connection). All node connections are allowed between layers. On the diagonal of the supra adjacency matrix, matrices are symmetrized, dediagonalized, semipositivized, and binarized. On the off-diagonal of the supra adjacency matrix, matrices are semipositivized and binarized.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultilayerBUD.gen.m]: same as top

File: braph2genesis/graphs/_MultilayerBUT.gen.m [hunk 7]
```diff
@@ -2,11 +2,11 @@
 MultilayerBUT < MultilayerWU (g, multilayer binary undirected with fixed thresholds graph) is a multilayer binary undirected graph with fixed thresholds.
 
 %%% ¡description!
-In a multilayer binary undirected with fixed thresholds (BUT) graph, the layers 
- are those of binary undirected (BU) multilayer graphs derived from the same 
+In a multilayer binary undirected with fixed thresholds graph (MultilayerBUT), the layers 
+ are those of binary undirected multilayer graphs derived from the same
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultilayerBUT.gen.m]: same as in MultilayerBUD

File: braph2genesis/graphs/_MultilayerBUT.gen.m [hunk 8]
```diff
@@ -145,12 +145,12 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+'In a multilayer binary undirected with fixed thresholds graph (MultilayerBUT), the layers are those of binary undirected multilayer graphs derived from the same weighted supra-adjacency matrix binarized at different thresholds. The supra-adjacency  matrix has a number of partitions equal to the number of thresholds. Layers within the binary undirected multilayer graphs could have different number of nodes with within-layer binary undirected edges. Edges can be either 0 (absence of connection) or 1 (existence of connection). All node connections are allowed between layers. On the diagonal of the supra adjacency matrix, matrices are symmetrized, dediagonalized, semipositivized, and binarized. On the off-diagonal of the supra adjacency matrix, matrices are semipositivized and binarized.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultilayerBUT.gen.m]: same as in MultilayerBUD

File: braph2genesis/graphs/_MultiplexBUD.gen.m [hunk 9]
```diff
@@ -2,8 +2,8 @@
 MultiplexBUD < MultiplexWU (g, binary undirected multiplex with fixed densities) is a binary undirected multiplex with fixed densities.
 
 %%% ¡description!
-In a binary undirected multiplex with fixed densities (BUD), the layers are 
- those of binary undirected (BU) multiplex graphs derived from the same 
+In a binary undirected multiplex with fixed densities graph (MultiplexBUD), the layers are
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultiplexBUD.gen.m]: same logic as in MultilayerBUD.
It should be:

In a **multiplex** binary undirected **graph** with fixed densities (MultiplexBUD), ...

File: braph2genesis/graphs/_MultiplexBUT.gen.m [hunk 10]
```diff
@@ -2,8 +2,8 @@
 MultiplexBUT < MultiplexWU (g, binary undirected multiplex with fixed thresholds) is a binary undirected multiplex with fixed thresholds.
 
 %%% ¡description!
-In a binary undirected multiplex with fixed thresholds (BUT), the layers are 
- those of binary undirected (BU) multiplex graphs derived from the same 
+In a binary undirected multiplex with fixed thresholds graph (MultiplexBUT), the layers are
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultiplexBUT.gen.m]: same logic as in MultiplexBUD

File: braph2genesis/graphs/_MultiplexBUT.gen.m [hunk 11]
```diff
@@ -133,12 +133,12 @@ those of binary undirected (BU) multiplex graphs derived from the same
...
 NAME (constant, string) is the name of the binary undirected multiplex with fixed thresholds.
 %%%% ¡default!
-'MultiplexBUT'
+'Multiplex Binary Undirected at fixed Thresholds'
 
 %%% ¡prop!
 DESCRIPTION (constant, string) is the description of the binary undirected multiplex with fixed thresholds.
 %%%% ¡default!
-'In a binary undirected multiplex with fixed thresholds (BUT), the layers are those of binary undirected (BU) multiplex graphs derived from the same weighted supra-connectivity matrices binarized at different thresholds. The supra-connectivity matrix has a number of partitions equal to the number of thresholds.'
+'In a binary undirected multiplex with fixed thresholds graph (MultiplexBUT), the layers are those of binary undirected multiplex graphs derived from the same weighted supra-connectivity matrices binarized at different thresholds. The supra-connectivity matrix has a number of partitions equal to the number of thresholds.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_MultiplexBUT.gen.m]: same logic as in MultiplexBUD

File: braph2genesis/graphs/_OrdMlBUD.gen.m [hunk 12]
```diff
@@ -1,11 +1,11 @@
 %% ¡header!
-OrdMlBUD < OrdMlWU (g, binary undirected multilayer with fixed densities) is an ordinal multilayer binary undirected with fixed densities.
+OrdMlBUD < OrdMlWU (g, ordinal multilayer binary undirected with fixed densities) is an ordinal multilayer binary undirected with fixed densities.
 
 %%% ¡description!
-In an ordinal binary undirected multilayer with fixed densities (BUD) graph, layers 
- consist of binary undirected (BU) multilayer graphs derived from the same 
+In an ordinal multilayer binary undirected with fixed densities graph (OrdMlBUD), layers
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMlBUD.gen.m]: same logic as in MultiplexBUD

should be:
In an ordinal multilayer binary undirected **graph** with fixed densities (OrdMlBUD)

File: braph2genesis/graphs/_OrdMlBUD.gen.m [hunk 13]
```diff
@@ -138,37 +138,37 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+'In an ordinal multilayer binary undirected with fixed densities graph (OrdMlBUD), layers consist of binary undirected multilayer graphs derived from the same weighted supra-connectivity matrices binarized at different densities. Layers within the binary undirected multilayer graphs could have different number of nodes with within-layer binary undirected edges. Edges can be either 0 (absence of connection) or 1 (existence of connection). The supra-connectivity matrix has a number of partitions equal to the number of densities. The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected. On the diagonal of the supra adjacency matrix, matrices are symmetrized, dediagonalized, semipositivized, and binarized. On the off-diagonal of the supra adjacency matrix, matrices are semipositivized and binarized.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMlBUD.gen.m]: same

File: braph2genesis/graphs/_OrdMlBUT.gen.m [hunk 14]
```diff
@@ -1,11 +1,11 @@
 %% ¡header!
-OrdMlBUT < OrdMlWU (g, binary undirected multilayer with fixed thresholds) is an ordinal multilayer binary undirected with fixed thresholds.
+OrdMlBUT < OrdMlWU (g, ordinal multilayer binary undirected with fixed thresholds) is an ordinal multilayer binary undirected with fixed thresholds.
 
 %%% ¡description!
-In an ordinal binary undirected multilayer with fixed thresholds (BUT) graph, layers 
- consist of binary undirected (BU) multilayer graphs derived from the same 
+In an ordinal multilayer binary undirected with fixed thresholds graph (OrdMlBUT), layers
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMlBUT.gen.m]: Same, should be

In an ordinal multilayer binary undirected **graph** with fixed thresholds (OrdMlBUT),

File: braph2genesis/graphs/_OrdMlBUT.gen.m [hunk 15]
```diff
@@ -132,37 +132,37 @@ within the binary undirected (BU) multilayer graphs could have different number
...
+'In an ordinal multilayer binary undirected with fixed thresholds graph (OrdMlBUT), layers consist of binary undirected multilayer graphs derived from the same weighted supra-connectivity matrices binarized at different thresholds. Layers within the binary undirected multilayer graphs could have different number of nodes with within-layer binary undirected edges. Edges can be either 0 (absence of connection) or 1 (existence of connection). The supra-connectivity matrix has a number of partitions equal to the number of thresholds. The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected. On the diagonal of the supra adjacency matrix, matrices are symmetrized, dediagonalized, semipositivized, and binarized. On the off-diagonal of the supra adjacency matrix, matrices are semipositivized and binarized.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMlBUT.gen.m]: same

File: braph2genesis/graphs/_OrdMxBUD.gen.m [hunk 16]
```diff
@@ -1,9 +1,9 @@
 %% ¡header!
-OrdMxBUD < OrdMxWU (g, binary undirected multiplex with fixed densities) is a ordinal multiplex binary undirected with fixed densities.
+OrdMxBUD < OrdMxWU (g, ordinal multiplex binary undirected with fixed densities) is a ordinal multiplex binary undirected with fixed densities.
 
 %%% ¡description!
-In an ordinal binary undirected multiplex with fixed densities (BUD), the layers are 
- those of binary undirected (BU) multiplex graphs derived from the same 
+In an ordinal multiplex binary undirected with fixed densities graph (OrdMxBUD), the layers are
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMxBUD.gen.m]: same

File: braph2genesis/graphs/_OrdMxBUD.gen.m [hunk 17]
```diff
@@ -127,37 +127,37 @@ those of binary undirected (BU) multiplex graphs derived from the same
...
+NAME (constant, string) is the name of the ordinal multiplex binary undirected with fixed densities.
 %%%% ¡default!
-'OrdMxBUD'
+'Ordinal Multiplex Binary Undirected at fixed Densities'
 
 %%% ¡prop!
-DESCRIPTION (constant, string) is the description of the ordinal binary undirected multiplex with fixed densities.
+DESCRIPTION (constant, string) is the description of the ordinal multiplex binary undirected with fixed densities.
 %%%% ¡default!
-'In an ordinal binary undirected multiplex with fixed densities (BUD), the layers are those of binary undirected (BU) multiplex graphs derived from the same weighted supra-adjacency matrix binarized at different densities. The supra-adjacency matrix has a number of partitions equal to the number of densities.The layers are connected in an ordinal fashion, where just consecutive layers are connected'
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMxBUD.gen.m:142]: same

File: braph2genesis/graphs/_OrdMxBUT.gen.m [hunk 18]
```diff
@@ -1,9 +1,9 @@
 %% ¡header!
-OrdMxBUT < OrdMxWU (g, ordinal multiplex binary undirected with fixed thresholds) is a binary undirected ordinal multiplex with fixed thresholds.
+OrdMxBUT < OrdMxWU (g, ordinal multiplex binary undirected with fixed thresholds) is a ordinal multiplex binary undirected with fixed thresholds.
 
 %%% ¡description!
-In a binary undirected ordinal multiplex with fixed thresholds (BUT), 
-all the layers consist of binary undirected (BU) multiplex graphs 
+In a ordinal multiplex binary undirected with fixed thresholds graphs (OrdMxBUT),
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMxBUT.gen.m]: same

File: braph2genesis/graphs/_OrdMxBUT.gen.m [hunk 19]
```diff
@@ -128,37 +128,37 @@ all the layers consist of binary undirected (BU) multiplex graphs
...
+NAME (constant, string) is the name of the ordinal multiplex binary undirected with fixed thresholds.
 %%%% ¡default!
-'OrdMxBUT'
+'Ordinal Multiplex Binary Undirected at fixed Thresholds'
 
 %%% ¡prop!
 DESCRIPTION (constant, string) is the description of the binary undirected multiplex with fixed thresholds.
 %%%% ¡default!
-'In a binary undirected ordinal multiplex with fixed thresholds (BUT),  all the layers consist of binary undirected (BU) multiplex graphs  derived from the same weighted supra-connectivity matrices  binarized at different thresholds. The supra-connectivity matrix has a number of partitions equal to the number of thresholds. The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.'
+'In a ordinal multiplex binary undirected with fixed threshold graph (OrdMxBUT),  all the layers consist of binary undirected multiplex graphs  derived from the same weighted supra-connectivity matrices  binarized at different thresholds. The supra-connectivity matrix has a number of partitions equal to the number of thresholds. The layers are connected in an ordinal fashion, i.e., only consecutive layers are connected.'
```
Comment (iok-ui) [inline braph2genesis/graphs/_OrdMxBUT.gen.m]: same

**Review by iok-ui — CHANGES_REQUESTED** [review]

**Pull request merged**